API Blueprint Specification.md

```diff
@@ -478,6 +490,232 @@ Name and description of the API
 
 ---
 
+<a name="def-authentication-section"></a>
+## Authentication section
+- **Parent sections:** none
+- **Nested sections:** [`1` Authentication schemes section](#def-authenticationschemes-section), [`1` Response section](#def-response-section)
+- **Markdown entity:** header
+- **Inherits from**: none
+
+#### Definition
+Defined by the `Authentication` keyword in Markdown header entity.
+
+    # Authentication
+
+#### Description
+Authentication settings for the whole API.
+
+This section **should** include one [Authentication schemes section](#def-authenticationschemes-section) and one [Response section](#def-response-section).
+
+The [Authentication schemes section](#def-authenticationschemes-section) defines all the types of authentication schemes supported by the API. The [Response section](#def-response-section) describes the error message returned in the case of failed authentication.
+
+##### Example
+
+    ## Authentication
+    This API supports different kinds of the authentication.
+
+    + Authentication Schemes
+        + ...
+
+    + Response 403 (application/json)
+
+            {
+              "message": "Authentication required",
```

Comment:
This message shouldn't be applicable to a 403 response. As per the HTTP specification ([rfc2616 section 10.4.4](http://www.ietf.org/rfc/rfc2616.txt)), where it states "authorization will **not** help".

> ### 10.4.4 403 Forbidden
> Authorization will not help and the request SHOULD NOT be repeated.

We should use a different example that doesn't go against RFC2616.